--- origsrc/oggvorbis-0.2/config.m4	2003-09-02 20:03:35.000000000 -0500
+++ src/oggvorbis-0.2/config.m4	2007-10-11 23:16:53.078125000 -0500
@@ -19,4 +19,5 @@
 
   PHP_NEW_EXTENSION(oggvorbis, oggvorbis.c,
 	$ext_shared,,)
+  PHP_SUBST(OGGVORBIS_SHARED_LIBADD)
 fi
--- origsrc/oggvorbis-0.2/oggvorbis.c	2003-09-10 18:42:07.000000000 -0500
+++ src/oggvorbis-0.2/oggvorbis.c	2007-10-11 23:14:53.031250000 -0500
@@ -199,7 +199,7 @@
 							vorbis_comment_add_tag(&(data->vcomment), key, tmpstr);
 							efree(tmpstr);
 							break;
-						default:
+						default:;
 							/* Do nothing with other types */
 					}
 				}
